tensorflow/tools/pip_package/setup.py

```diff
@@ -60,6 +60,11 @@
 # pylint: disable=line-too-long
 CONSOLE_SCRIPTS = [
     'saved_model_cli = tensorflow.python.tools.saved_model_cli:main',
+    # We need to keep the TensorBoard command, even though the console script
+    # is now declared by the tensorboard pip package. If we remove the
```

Comment:
nit: I'd write either "the TensorBoard pip package" or "the tensorflow-tensorboard pip package" (because `tensorflow-tensorboard` is the pip package for the TensorBoard application).